doc/rpc/rescanblockchain.md

```diff
@@ -0,0 +1,67 @@
+# `rescanblockchain`
+
+Sends a request to the node for rescan the blockchain searching for transactions related to the wallets cached addresses.
+
+## Arguments
+
+* `start_height` (numeric, optional, default=`0`): The initial block height to start the blockchain rescan.
+
+* `stop_height` (numeric, optional, default=`0`): The height limit to stop the rescanning.
+
+* `use_timestamp` (boolean flag, optional, default=`false`): When present in the command the provided values will be treated as UNIX timestamps. These timestamps do not need to be directly from a block and can be used for finding addresses and UTXOs from meaningful timestamp values.
+
+* `confidence` (string, optional, default=`medium`): In cases where `use_timestamp` is set, tells how much confidence the user wants for finding its addresses from this rescan request, a higher confidence will add more lookback seconds to the targeted timestamp and rescanning more blocks.
+
+Under the hood this uses an [Exponential distribution](https://en.wikipedia.org/wiki/Exponential_distribution) [cumulative distribution function (CDF)](https://en.wikipedia.org/wiki/Cumulative_distribution_function) where the parameter $\lambda$ (rate) is $\frac{1}{600}$ (1 block every 600 seconds, 10 minutes).
+  The supplied string can be one of:
+  
+  - `high`: 99% confidence interval.
+  - `medium` (default): 95% confidence interval.
+  - `low`: 90% confidence interval.
+  - `none`: Doesnt apply any lookback seconds.
+
+## Returns
+
+### Ok Response
+
+When the rescan request sucessfully starts you will receive an Ok(true).
```

Comment:
After the rescan finishes, you'll see a log message telling all the blocks that may have any transactions to your wallet